egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -19,36 +31,43 @@ hidden_dim=1024
 recurrent_projection_dim=256
 non_recurrent_projection_dim=256
 chunk_width=20
-chunk_left_context=20
-clipping_threshold=5.0
+chunk_left_context=40
```

Comment:
Yes this was slightly better.

Vijay

On Tue, Oct 6, 2015 at 4:39 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/nnet3/run_lstm.sh
> <https://github.com/kaldi-asr/kaldi/pull/214#discussion_r41318798>:
>
> > @@ -19,36 +31,43 @@ hidden_dim=1024
> >  recurrent_projection_dim=256
> >  non_recurrent_projection_dim=256
> >  chunk_width=20
> > -chunk_left_context=20
> > -clipping_threshold=5.0
> > +chunk_left_context=40
>
> we're using chunk-left-context=40 now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/214/files#r41318798>.
>